Doc/howto/clinic.rst

```diff
@@ -162,28 +162,20 @@ For the sake of clarity, here's the terminology we'll use with Argument Clinic:
 Tutorial
 ========
 
-The best way to get a sense of how Argument Clinic works is to
-convert a function to work with it.  Here, then, are the bare
-minimum steps you'd need to follow to convert a function to
-work with Argument Clinic.  Note that for code you plan to
-check in to CPython, you really should take the conversion farther,
-using some of the :ref:`advanced concepts <clinic-howtos>`
-you'll see later on in the document,
-like :ref:`clinic-howto-return-converters`
-and :ref:`clinic-howto-self-converter`.
-But we'll keep it simple for this walkthrough so you can learn.
-
-First, make sure you're working with a freshly updated checkout
-of the CPython trunk.
-
-Next, find a Python builtin that calls either :c:func:`PyArg_ParseTuple`
+In this tutorial, you will convert a C function from having custom
+:ref:`argument parsing code <arg-parsing>`
+to using Argument Clinic to generate that boilerplate code.
```

Comment:
I feel this sentence could be better, maybe something like:
```suggestion
In this tutorial, you will learn how to use Argument Clinic to generate
:ref:`argument parsing code <arg-parsing>` for a function that
was using custom code.
```

The current sentence uses the imperative form to tell you what to do, without telling you why, in a way that sounds a bit harsh.  "Converting a function" seems something an howto would cover, whereas "learn how to use AC (to convert functions)" seems a better fit for the tutorial.

In addition, using "we" ("*we will learn how*") sounds "friendlier" to me, even thought technically only the reader is learning. This might require a refactoring of the whole tutorial, even though I see there are some instances of "we" later.  I don't remember if diataxis has any preference regarding this.